src/multimeasurerest.ts

```diff
@@ -91,34 +74,38 @@ export class MultiMeasureRest extends Element {
    */
   constructor(numberOfMeasures: number, options: MultimeasureRestRenderOptions) {
     super();
+    const fontSize = options.numberGlyphPoint ?? Tables.lookupMetric('MultiMeasureRest.fontSize'); // same as TimeSignature.
+    this.textFont.size = fontSize;
 
     this.numberOfMeasures = numberOfMeasures;
+    this.text = '';
+    const text = `${this.numberOfMeasures}`;
+    for (let i = 0; i < text.length; i++) {
+      this.text += String.fromCodePoint(0xe080 + text.charCodeAt(i) - '0'.charCodeAt(0)) /*timeSigN*/;
+    }
```

Comment:
Can this be rewritten as:

```ts
const t = `${this.numberOfMeasures}`;
for (let digit of t) {
  // 0xe080 is timeSig0. We calculate the code point for timeSigN to assemble the digits via SMuFL glyphs.
  this.text += String.fromCodePoint(0xe080 + Number(digit));
}
```

Let me know if it works!

Reply:
it works